ports/mimxrt10xx/supervisor/flexspi_nor_flash_ops.c

```diff
@@ -12,7 +12,7 @@
 #include "boards/flash_config.h"
 #include "supervisor/linker.h"
 
-status_t PLACE_IN_ITCM(flexspi_nor_write_enable)(FLEXSPI_Type * base, uint32_t baseAddr)
+status_t PLACE_IN_ITCM(static flexspi_nor_write_enable)(FLEXSPI_Type * base, uint32_t baseAddr)
```

Comment:
I think static needs to be first here. Like below. The macro uses the name in the section name too.